src/coreclr/inc/corjit.h

```diff
@@ -144,6 +144,14 @@ enum CheckedWriteBarrierKinds {
 
 #include "corjithost.h"
 
+#if !defined(_MSC_VER) && !defined(__stdcall)
```

Comment:
Can we delete stdcall and just depend on the build defaults? This is not a public header, so we do not need to make it work in the presence of arbitrary build defaults.

Reply:
Is it safe to assume that all users of the interface are compiled the same way (i.e., all assume stdcall by default, or all assume cdecl)? On Windows it was always best practice to be explicit about the calling convention of cross-module APIs.

Reply:
JIT/EE interface is not a public interface. All users of this interface live in this repo and are under our control. We compile everything with same compiler flags.